aptos-move/e2e-move-tests/src/tests/transaction_context.data/pack/sources/transaction_context_test.move

```diff
@@ -0,0 +1,54 @@
+module 0x1::transaction_context_test {
+    use std::option::{Self, Option};
+    use aptos_framework::transaction_context;
+    use std::features;
+
+    /// Since tests in e2e-move-tests/ can only call entry functions which don't have return values, we must store
+    /// the results we are interested in (i.e., the chain ID) inside this (rather-artificial) resource, which we can
+    /// read back in our e2e-move-tests/ test.
+    struct TransactionContextStore has key {
+        sender: address,
+        secondary_signers: vector<address>,
+        fee_payer_opt: Option<address>,
+        max_gas_amount: u64,
+    }
+
+    /// Called when the module is first deployed at address `signer`, which is set to 0x1 (according to the `module 0x1::chain_id_test` line above).
+    fun init_module(sender: &signer) {
+        move_to(sender,
+            TransactionContextStore {
+                sender: @0x0,
```

Comment:
Does this address mean anything? Perhaps replace with a constant naming what this represents :)